pilot/pkg/networking/grpcgen/cds.go

```diff
@@ -129,6 +130,13 @@ func (b *clusterBuilder) build() []*cluster.Cluster {
 	var defaultCluster *cluster.Cluster
 	if b.filter.Contains(b.defaultClusterName) {
 		defaultCluster = edsCluster(b.defaultClusterName)
+		if b.svc.Attributes.Labels[features.PersistentSessionLabel] != "" {
+			// see core/v1alpha3/cluster.go
+			defaultCluster.CommonLbConfig.OverrideHostStatus = &core.HealthStatusSet{
+				Statuses: []core.HealthStatus{core.HealthStatus_HEALTHY, core.HealthStatus_UNHEALTHY,
+					core.HealthStatus_DRAINING, core.HealthStatus_UNKNOWN, core.HealthStatus_DEGRADED},
```

Comment:
Why include UNHEALTHY and DEGRADED? I thought for session draining (when persistence is enabled) only DRAINING should be added (in addition to HEALTHY and UNKNOWN)

Reply:
DEGRADED - based on Kuat's feedback. We don't generate that state, but if we did ( or EDS federation is used ) it seems reasonable to use it.
UNHEALTHY - we also don't generate it, but most of the EnvoyFilter-based examples I've seen from users who tried persistent sessions had it enabled. 

We can adjust it in a separate PR - for now it's a no-op.

Reply:
Yeah, because it's a default - so we have to respect that as a matter of compatibility (we never set this flag before). Can be done later - but there's little problem with it now.

Reply:
> UNHEALTHY - we also don't generate it, but most of the EnvoyFilter-based examples I've seen from users who tried persistent sessions had it enabled.

I think that was before DRAINING was available and supported, so shouldn't be necessary now. 

I tend to agree with @sanjaypujare comments